nasl/nasl_cmd_exec.c

```diff
@@ -94,14 +95,24 @@ pread_streams (int fdout, int fderr)
 }
 
 /** @todo Supspects to glib replacements, all path related stuff. */
+/**
+ * @brief Spawn a process
+ *
+ * @param[in] lexic   Lexical context of NASL interpreter.
+ * @param[in] cmd Command to be run.
+ * @param[in] argv List of arguments.
+ * @param[in] drop_privileges Owner of the spawned process.
```

Comment:
The name `drop_privileges` for the parameter sounds more like one which accepts a TRUE/FALSE, not a user as a string. Maybe something like e.g. the following could make sense:

```suggestion
 * @param[in] drop_privileges_user Owner of the spawned process.
```